src/analyze/analyze.c

```diff
@@ -1832,6 +2010,50 @@ static int do_security(int argc, char *argv[], void *userdata) {
         return analyze_security(bus, strv_skip(argv, 1), 0);
 }
 
+static int parse_arg_color_style(DotArgColorStyle color_or_style, const char *arg_dot_color_style) {
+        char delim[] = ":";
+        const char *p;
+        int r;
+
+        p = arg_dot_color_style;
+        _cleanup_free_ char *word = NULL;
+
+        r = extract_first_word(&p, &word, delim, 0);
+        if (r == 0)
+                return r;
```

Comment:
if i pass the empty string this should generate a parse error, and not be ignored

Reply:
done